samples/qa-virtualgridlist/package.json

```diff
@@ -22,11 +22,12 @@
     "theme": "sandstone"
   },
   "dependencies": {
-    "@enact/core": "^4.9.0",
-    "@enact/i18n": "^4.9.0",
+    "@enact/core": "^5.0.0-alpha.1",
+    "@enact/i18n": "^5.0.0-alpha.1",
     "@enact/sandstone": "../../",
-    "@enact/spotlight": "^4.9.0",
-    "@enact/ui": "^4.9.0",
+    "@enact/spotlight": "^5.0.0-alpha.1",
+    "@enact/ui": "^5.0.0-alpha.1",
+    "@reduxjs/toolkit": "^2.2.2",
```

Comment:
Please remove this. We've removed this from https://github.com/enactjs/sandstone/pull/1572.

Reply:
ok